qiskit_dynamics/perturbation/multiset_utils.py

```diff
@@ -46,15 +46,51 @@ def _multiset_to_sorted_list(multiset: Multiset) -> List:
     return sorted_list
 
 
-def _sorted_multisets(multisets: List[Multiset]) -> List[Multiset]:
-    """Sort in non-decreasing order according to:
-
-    ms1 <= ms2 if len(ms1) < len(ms2), or if
-    len(ms1) == len(ms2) and if
-    str(_multiset_to_sorted_list(ms1)) <= str(_multiset_to_sorted_list(ms2)).
+class _MultisetSort:
```

Comment:
nit: consider renaming to _MultisetSortKey

Reply:
Changed.